x/oracle/keeper/owasm.go

```diff
@@ -12,6 +12,13 @@ import (
 	"github.com/bandprotocol/chain/x/oracle/types"
 )
 
+// 1 cosmos gas is equal to 7 owasm gas
+const gasConversionFactor = 7
+
+func convertToOwasmGas(cosmos uint64) uint32 {
+	return uint32(cosmos * gasConversionFactor)
```

Comment:
can we use cosmos type `Uint64` for safe multiplication

Reply:
Prepare and execute functions accept uint32 so we have to cast it anyway